### PR TITLE
Added new useful methods to Pokemon.java

### PR DESCRIPTION
**Pull request by gionata-bisciari**
+getStaminaInPercentage():int
+getCPInPercentageActualPlayerLevel():int
+getCPInPercentageMaxPlayerLevel:int
+getIvInPercentage():double

### CODE REVIEW

File: library/src/main/java/com/pokegoapi/api/pokemon/Pokemon.java [hunk 1]
```diff
@@ -403,4 +404,36 @@ public boolean isFainted() {
...
+	 * @throws NoSuchItemException if threw from {@link #getMaxCp()}
+	 */
+	public int getCPInPercentageMaxPlayerLevel() throws NoSuchItemException{
+		return ((getCp()*100)/getMaxCp());
+	}
+	
+	/**
+	 * @return IV in percentage
+	 */
+	public double getIvInPercentage(){
```
Comment (FabianTerhorst) [inline library/src/main/java/com/pokegoapi/api/pokemon/Pokemon.java]: Add Space between ) and {

**Comment by AbandonedCart** [comment]
That's a nice way to interpret those values, especially when comparing them.
Too bad at anything below 35%, it can't just return `NAME fails as a Pokemon`

Out of curiosity, why does the final one return a double?
As far as I know, Math.floor will already return an int.

**Comment by gionata-bisciari** [comment]
ivs
it returns a double with two decimal places

**Comment by AbandonedCart** [comment]
My mistake. Missed the second outer parenthesis for the / 100

**Comment by FabianTerhorst** [comment]
Please fix the style warnings.

**Comment by gionata-bisciari** [comment]
what style warnings?

**Comment by FabianTerhorst** [comment]
When you run the gradle task on your project.

**Comment by FabianTerhorst** [comment]
I still don´t know why travis doesn´t print them.

**Comment by gionata-bisciari** [comment]
I haven't seen it infact, I'll check

**Comment by FabianTerhorst** [comment]
👍

**Comment by Grover-c13** [comment]
yeah i really wish we could get travis to fail on checkstyle :(

**Comment by gionata-bisciari** [comment]
Is now everything ok?

**Pull request merged**